lib/LINE/Bot/Audience.pm

```diff
@@ -52,6 +63,11 @@ __END__
 
 LINE::Bot::Audience
 
+=head1 C<< rename_audience({ description => "...", audience_group_id => "..." }) >>
+Renames an existing audience.
```

Comment:
A blank line is required after `head1` (or all header marks) otherwise `perldoc` wrongly render this part.

I'll just merge PRs for now and apply this documentation fixes latter.